go.sum

```diff
@@ -16,13 +16,17 @@ github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7 h1:YoJbenK9C6
 github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7/go.mod h1:z4/9nQmJSSwwds7ejkxaJwO37dru3geImFUdJlaLzQo=
 github.com/acomagu/bufpipe v1.0.3 h1:fxAGrHZTgQ9w5QqVItgzwj235/uYZYgbXitB+dLupOk=
 github.com/acomagu/bufpipe v1.0.3/go.mod h1:mxdxdup/WdsKVreO5GpW4+M/1CE2sMG4jeGJ2sYmHc4=
+github.com/agext/levenshtein v1.2.1/go.mod h1:JEDfjyjHDjOF/1e4FlBE/PkbqA9OfWu2ki2W0IB5558=
```

Comment:
is the a reason this files changed?
I don't see changes in packages/imports 🤔

Reply:
when building it auto-updated, I've always found go.sum a bit weird. It's now updated I belive with an additional import.

Let me know your thoughts and I can have a look at solving conflicts there.

Reply:
Could you run `go mod tidy` before committing - it removes not used hashsums and should cleanup this file a bit.